src/components/admissionProcedure/AdmissionProcedure.jsx

```
@@ -316,91 +457,150 @@ export default function AdmissionProcedure() {
-                      className="grid md:grid-cols-2 lg:grid-cols-3 gap-6"
+                      className="grid md:grid-cols-2 gap-6"
                       variants={containerVariants}
                     >
                       {freshmenSteps.map((step, index) => (
                         <motion.div
                           key={index}
-                          className={`bg-white p-6 rounded-xl shadow-md cursor-pointer border border-gray-200 hover:border-msu-maroon ${hoveredStep === index ? 'ring-2 ring-msu-maroon' : ''}`}
+                          className={`bg-white p-6 rounded-xl shadow-sm cursor-pointer border border-gray-200 hover:border-msu-maroon transition-all ${hoveredStep === index ? 'ring-2 ring-msu-maroon' : ''}`}
                           variants={itemVariants}
                           whileHover="hover"
                           variants={scaleUp}
```

Biome (1.9.4)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)